api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -317,11 +331,16 @@ func (c *imageBuilder) createKanikoJobSpec(project mlp.Project, model *models.Mo
 		labelOrchestratorName: "merlin",
 	}
 
+	baseImageTag, ok := c.config.BaseImages[version.PythonVersion]
```

Comment:
how about if the user give 3.7 instead of 3.7.* for python version?

Reply:
the value is controlled by us, not by the user. User's part is managed from python sdk side already.